browser/resources/extensions/BUILD.gn

```diff
@@ -1,3 +1,8 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
please drop this change from the PR since it isn't relevant

Reply:
This block CI with presubmit checks